monGARS/core/llm_integration.py

```
@@ -400,30 +400,90 @@ async def _ensure_adapter_metadata(self) -> dict[str, str] | None:
+                    "adapter_version": payload.get("version"),
+                    "task_type": task_type,
+                },
+            )
+            return payload
+
+        return metadata
+
+    def _load_reasoning_adapter_payload(self) -> dict[str, str] | None:
+        try:
+            manifest = load_manifest(self.adapter_registry_path)
+        except Exception as exc:  # pragma: no cover - defensive logging
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)